api/users/serializers.py

```diff
@@ -117,6 +122,21 @@ def get_absolute_url(self, obj):
             'version': self.context['request'].parser_context['kwargs']['version']
         })
 
+    def get_node_count(self, obj):
+        return obj.nodes.count()
+
+    def get_quickfiles_count(self, obj):
+        return QuickFilesNode.objects.get(contributor__user__id=obj.id).files.filter(type='osf.osfstoragefile').count()
+
+    def get_registration_count(self, obj):
+        return Registration.objects.filter(contributor__user__id=obj.id).count()
```

Comment:
Same here, double check to make sure this matches what's returned by `UserRegistrations`, I imagine you'll have to exclude more things.